src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
```

Comment:
Consider using a constant for the 'perDay' cost to make it clear that this value is not meant to change.